tests/pyanaconda_tests/user_test.py

```diff
@@ -90,3 +96,43 @@ def length_test(self):
 
         self._assert_username("f" * 32, True)
         self._assert_username("f" * 1, True)
+
+
+class GroupNameTests(UserNameTests):
+
+    def _check_name(self, name):
+        return check_groupname(name)
+
+    def reserved_names_test(self):
+        """There are no reserved names for groups."""
+        self._assert_name("root", True)
+        self._assert_name("home", True)
+        self._assert_name("system", True)
+
+class GroupListTests(GroupNameTests):
+
+    def _check_name(self, name):
+        return check_grouplist(name)
+
+    def grouplist_test(self):
+        """Test a simple list of groups."""
+        self._assert_name("", True)
+        self._assert_name("foo", True)
+        self._assert_name(" foo", True)
+        self._assert_name(" \tfoo", True)
+        self._assert_name("foo ", True)
+        self._assert_name("foo \t", True)
+        self._assert_name("  foo  ", True)
+        self._assert_name("foo,bar", True)
+        self._assert_name("foo, bar", True)
+        self._assert_name("  foo,    bar", True)
+        self._assert_name("foo, bar, xxx", True)
+
+        self._assert_name(",", False)
```

Comment:
I'm missing test for `0` which is mentioned in the bug.

Reply:
Fixed.